addons/base_vat/models/res_partner.py

```diff
@@ -363,3 +363,74 @@ def check_vat_al(self, vat):
             return stdnum.al.vat.is_valid(vat)
         except ImportError:
             return True
+
+    def check_vat_nl(self, vat):
```

Comment:
You can copy most of the code from the library : https://github.com/arthurdejong/python-stdnum/blob/087c668560148de86084b247c4dffafb19f5c3fe/stdnum/nl/btw.py#L55

It's only the last part that it's different.